skin/frontend/base/default/algoliasearch/algoliasearch.css

```diff
@@ -1211,4 +1211,10 @@ a.ais-current-refined-values--link:hover
 
 #algolia-right-container .infos.hidden {
 	display: none;
+}
+
+.algolia-clearfix:after {
+	content:"";
+	display:block;
```

Comment:
On the shared link, they use `display:table`, it probably addresses some edge case so I think it is better to use it instead of `block`.